### ISSUE TITLE
Imported JIG: Words too large, voiceover missing on questions, and no background music

### ISSUE THREAD

**Issue by corinnewo** [ji-devs/ji-cloud#2546]
Game PURIM FUN by Sammy:
Ji Tap: https://jitap.net/activities/gzd/play/
Jigzi: https://jigzi.org/jig/play/7efa89a6-69c2-11ec-ab82-33693acdeea0?direction=ltr&display_score=false&track_assessments=false&drag_assist=false&is_student=false&draft=false
**1. Texts are much larger than on Ji Tap - don't fit on screen:** 
![image](https://user-images.githubusercontent.com/16290361/158324603-60f2797e-0058-46a3-89cb-cb0c89f97e95.png)

![image](https://user-images.githubusercontent.com/16290361/158324558-f97843e5-6421-4789-9968-22df174cbef1.png)
On Ji Tap, looks like this: 
![image](https://user-images.githubusercontent.com/16290361/158324817-da372890-917e-45d5-80f3-25531f1f5047.png)

**2. Questions are not playing:**  Screen 04 - no question voiceover is playing, so can't even answer the questions

**3. Background music not playing** - I know that this is in process, but we really need to implement the background music in the JIGs ASAP.
